internal/pkg/template/diff/override_test.go

```diff
@@ -234,6 +234,12 @@ ImageId: !FindInMap
 				}
 			},
 		},
+		"diff in Fn::GetAtt vs !GetAtt when comparing scalar to scalar both with tags": {
```

Comment:
aren't old and curr the same thing 🤔 why is it useful?

Reply:
I should change the test case name to "no diff...." because I want to make sure no diff is being produced.